src/Hl7.Fhir.Core/Rest/FhirClient.cs

```diff
@@ -19,15 +18,16 @@ namespace Hl7.Fhir.Rest
 {
     public partial class FhirClient : BaseFhirClient
     {
-//disables warning that OnBeforeRequest and OnAfterResponse are never used.
+        //disables warning that OnBeforeRequest and OnAfterResponse are never used.
 #pragma warning disable CS0067
 
         /// <summary>
         /// Creates a new client using a default endpoint
         /// If the endpoint does not end with a slash (/), it will be added.
         /// </summary>
         /// <remarks>
-        /// If the messageHandler is provided then it must be disposed by the caller
+        /// If the messageHandler, or httpClient is provided then it must be disposed by the caller
+        /// Only one of the messageHandler or httpClient can be provided
```

Comment:
```suggestion

```
```suggestion
        /// Only one of the messageHandler or httpClient can be provided
```

Reply:
Updated the comments on the 4 constructors